sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -216,22 +216,15 @@ bool gnrc_rpl_parent_remove(gnrc_rpl_parent_t *parent)
     gnrc_rpl_dodag_t *dodag = parent->dodag;
 
     if (parent == dodag->parents) {
-        fib_remove_entry(&gnrc_ipv6_fib_table,
-                         (uint8_t *) ipv6_addr_unspecified.u8,
-                         sizeof(ipv6_addr_t));
+        gnrc_ipv6_nib_ft_del(&ipv6_addr_unspecified, 0);
```

Comment:
Note to self: this might cause problems with address registration => investigate.